lib/core/src/lib/viewer/components/viewer.component.html

```diff
@@ -61,7 +61,9 @@
                     </button>
                 </div>
 
-                <ng-content select="adf-viewer-toolbar-actions"></ng-content>
+                <ng-container *ngIf="showViewerToolbarActions">
+                    <ng-template [ngTemplateOutlet]="viewerToolbarActionsTemplate"></ng-template>
```

Comment:
probably this could be shortened to:
`<ng-container *ngIf="showViewerToolbarActions" [ngTemplateOutlet]="viewerToolbarActionsTemplate"></ng-container>`

(but I'm writing it without having tested it so please make sure that this would work)

Reply:
I restored the previous line `<ng-content select="adf-viewer-toolbar-actions"></ng-content>` because I project the content from the parent component https://github.com/Alfresco/alfresco-ng2-components/pull/7992/commits/06eb3717576ebe5243dbb922c8eaad8c2436a166#diff-1e6545055d3b5bad8e89f7015feb1fef5251debe688433aeb3b21b10d09a8195R31, so we don't need to use ngTemplateOutlet